java-client/src/main/java/co/elastic/clients/transport/rest_client/RestClientOptions.java

```diff
@@ -181,6 +192,11 @@ public TransportOptions.Builder onWarnings(Function<List<String>, Boolean> liste
             return this;
         }
 
+        @Override
```

Comment:
this is what I'm not liking about this solution, because this method does nothing, but I don't think it can be avoided